### PR TITLE
Refactored; Added analysis script

### PR DESCRIPTION
**Pull request by FarazzShaikh**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 output
+*.csv
+*.png
```
Comment (IamShubhamGupto) [inline .gitignore:3]: wont this mess with images under samples/ ?

File: analysis/plot.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import matplotlib.pyplot as plt
+import numpy as np
+  
+data = np.genfromtxt('analysis/analysis.csv', delimiter=',', names=['type', 'dt', 'N'])
+
+plt.plot(data['N'], data['dt'], color='r', label='the data')
```
Comment (IamShubhamGupto) [inline analysis/plot.py:6]: isnt this already doing whats required ?

Comment (FarazzShaikh) [reply analysis/plot.py:6]: yeah idk what im doing in python 😂

Comment (FarazzShaikh) [reply analysis/plot.py:6]: just make it look good like this one

plot

**Pull request merged**